tslint.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": [
+    "tslint-config-standard",
+    "tslint-config-prettier",
+    "tslint-react"
+  ],
+  "rules": {
+    "space-before-function-paren": false
+  }
```

Comment:
https://palantir.github.io/tslint/rules/no-default-export/

Especially as there is no TS right now